### PR TITLE
feat: add CI for MySQL

### PR DESCRIPTION
**Pull request by jiashenC**

### CODE REVIEW

**Comment by jiashenC** [comment]
I will move the CI to `staging` and `master` only once it is approved. For now, just make sure CI runs fine.

File: evadb/third_party/databases/mysql/mysql_handler.py [hunk 1]
```diff
@@ -109,3 +110,23 @@ def execute_native_query(self, query_string: str) -> DBHandlerResponse:
...
+            "float": float,
+            "double": float,
+            # Add more mappings as needed
+        }
+
+        if mysql_type in mapping:
+            return mapping[mysql_type]
+        else:
+            raise Exception(
+                f"Unsupported column {mysql_type} encountered in the mysql table. Please raise a feature request!"
```
Comment (gaurav274) [inline evadb/third_party/databases/mysql/mysql_handler.py:131]: Can you also add the link to our Slack in this message (https://evadb.ai/community)?

File: test/third_party_tests/test_native_executor.py [hunk 2]
```diff
@@ -186,10 +186,6 @@ def test_should_run_query_in_mysql(self):
         self._execute_native_query()
         self._execute_evadb_query()
 
-        # Test error.
-        self._raise_error_on_multiple_creation()
```
Comment (gaurav274) [inline test/third_party_tests/test_native_executor.py:190]: Why are we removing it?

File: evadb/third_party/databases/mysql/mysql_handler.py (hunk 1 above)
Comment (gaurav274) [reply evadb/third_party/databases/mysql/mysql_handler.py:131]: Add `self.connection = None` in the base class. Else it throws property not found in some cases.

File: test/third_party_tests/test_native_executor.py (hunk 2 above)
Comment (jiashenC) [reply test/third_party_tests/test_native_executor.py:190]: This test is actually invariant on which database engine we use. I currently just test on `Postgres`, so we should skip here.

File: evadb/third_party/databases/mysql/mysql_handler.py (hunk 1 above)
Comment (jiashenC) [reply evadb/third_party/databases/mysql/mysql_handler.py:131]: What do you mean by the link?

**Review by xzdandy — APPROVED** [review]

**Pull request merged**